### ISSUE TITLE
Version 3.3.0 breaks IE11

### ISSUE THREAD

**Issue by Richacinas** [moroshko/autosuggest-highlight#23]
I have done my research and found out that the latest version of this library (v3.3.0) is breaking Internet Explorer.

In our case, IE11.

The problem is that with this latest version there is some ES6 code coming (like `const`).

**Comment by dmitry-kovalev** [comment]
Hi
Please check on latest 3.3.1 version.

**Comment by Richacinas** [comment]
Hi again!

Thanks for the prompt answer.

I tried again with the 3.3.1 version, and the error is different. It seems to be located in the `remove-accents` library code:

```
function extend(subject, baseObject) {
  subject = subject || {};
THIS LINE IS BREAKING THE BUNDLE IN IE11 --->  Object.keys(subject).forEach((key) => {
    baseObject[key] = !!subject[key];
  });
  return baseObject;
}
```

**Comment by dmitry-kovalev** [comment]
It's quite strange because of 3.2.0 version contains same part of a code
```
function extend(subject, baseObject) {
  subject = subject || {};
  Object.keys(subject).forEach(function(key) {
    baseObject[key] = !!subject[key];
  });
  return baseObject;
}
```
And as I see this code was added exactly in 3.2.0.
What worked version are you using now?

PS. I tried to build the latest version with and without polyfills, but it increase bundle size from 6K to 60K.

**Comment by Richacinas** [comment]
We are using 3.1.1.. Wow, that's a massive increase of bundle size. I'm not sure how to go from there, to be honest.

**Comment by Richacinas** [comment]
I tried, but I'm getting the following error:

` Module not found: Error: Can't resolve 'autosuggest-highlight/ie11/parse' in`...

Checking on the folders created on `node_modules`, I don't see anything about `ie11`:

![image](https://user-images.githubusercontent.com/7046504/179195419-de56c987-cab0-4ab8-9515-7f7cf9019ed6.png)

**Comment by dmitry-kovalev** [comment]
Hi
Sorry for delaying with answer.
I've published version 3.3.3 with ie11 modules included.
